benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -448,17 +433,16 @@ msgstr "Select a benefit option"
 
 msgid "eligibility.pages.index.headline"
 msgstr ""
-"Connect your bank card to your public transit discount with Cal-ITP Benefits"
+"Connect your bank card to your public transit discount with Cal-ITP Benefits."
 
 #, python-format
 msgid "eligibility.pages.index.p[0]%(info_link)s"
 msgstr ""
-"You can tap your credit or debit card when you board, and your discount will "
+"You can tap your credit or debit card when you board an MST bus and your discounted fare will "
```

Comment:
This sentence (or at least part of it) may need to be put on the `EligibilityVerifier` model since it mentions "an MST bus". Also, I see that the copy from Figma removed the comma from `board,`, but I think we need it after `MST bus` to be grammatically correct.

Reply:
Will separate this line out into the model!

And will add the `,`.